app/views/layouts/application.html.haml

```diff
@@ -14,3 +14,4 @@
     = render 'layouts/flash'
     = yield
     %script{:src => "https://code.jquery.com/jquery-2.1.1.min.js", :type => "text/javascript"}
+    %script{:src => "https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.0/jquery.min.js"}
```

Comment:
Line is too long. [89/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace